checks/common.go

```diff
@@ -191,3 +199,45 @@ func getNextRuntime(canary v1.Canary, lastRuntime time.Time) (*time.Time, error)
 	t := lastRuntime.Add(time.Duration(canary.Spec.Interval) * time.Second)
 	return &t, nil
 }
+
+func template(ctx *context.Context, template v1.Template) (string, error) {
+	if template.Template != "" {
+		tpl := gotemplate.New("")
+
+		funcs := gomplate.Funcs(nil)
+		funcs["humanizeBytes"] = mb
```

Comment:
Can we move these functions to commons/template alongside jsonpath and markdownParser from kommons/ktemplate - we then a have a common base of functions to work with ?